dep11/generator.py

```diff
@@ -274,6 +296,8 @@ def handle_error(e):
 
             log.info("Completed metadata extraction for suite %s/%s" % (suite_name, component))
 
+        langpacks.cleanup()
```

Comment:
More pythonic code and ensures the cleanup is called even in crashes (like a `finally:`), not sure how much that matters in this case.